### PR TITLE
#7 Task. Alexander Zenchyk

### PR DESCRIPTION
**Pull request by zeeenjaaa**
https://travis-ci.com/github/zeeenjaaa/js-assignments/builds/175733823

### CODE REVIEW

**Comment by vramaniuk** [comment]
I'll close this PR , but in future you should resolve conflicting files (especially in real project)

File: task/07-yield-tasks.js [hunk 1]
```diff
@@ -145,7 +205,8 @@ function* mergeSortedSequences(source1, source2) {
  *   Most popular implementation of the logic in npm https://www.npmjs.com/package/co
  */
 function async(generator) {
-    throw new Error('Not implemented');
+  let co = require('co');
+  return co(generator);
```
Comment (ilya4321) [inline task/07-yield-tasks.js]: The idea is to implement alternative solution based on generators and Promises, I'll reopen the PR

Comment (zeeenjaaa) [reply task/07-yield-tasks.js]: ok, done.

**Comment by zeeenjaaa** [comment]
new build:
https://travis-ci.com/github/zeeenjaaa/js-assignments/builds/179950838

**Comment by IlyaMokin** [comment]
Accepted

**Pull request closed without merging**